lib/entities/statistics.rb

```diff
@@ -0,0 +1,35 @@
+class Statistics
+  include Database
+
+  def winners(base)
+    data = multi_sort(base)
+    array_rows = to_table(data)
+    table(array_rows)
+  end
+
+  private
+
+  def to_table(data)
```

Comment:
redo. you mustn't push it to array

Reply:
For Terminal::Table.new all lines have to be in array.